conf/pawsey_setonix.config

```diff
@@ -0,0 +1,29 @@
+// Pawsey Setonix nf-core configuration profile
+params {
+  config_profile_description = 'Pawsey Setonix HPC profile provided by nf-core/configs'
+  config_profile_contact = 'Sarah Beecroft (@SarahBeecroft), Georgie Samaha (@georgiesamaha)'
+  config_profile_url = 'https://support.pawsey.org.au/documentation/display/US/Setonix+User+Guide'
+}
+
+// Enable use of Singularity to run containers
+singularity {
+  enabled = true
+  autoMounts = true
+  autoCleanUp = true
+}
+
+// Submit up to 256 concurrent jobs (Setonix work partition max)
+executor {
+  queueSize = 1024
```

Comment:
Should this be 256 based on the comment?

Reply:
Can have 1024 max queued, but only 256 will run concurrently. Suggestion from Pawsey colleague.